### ISSUE TITLE
querying other NED tables

### ISSUE THREAD

**Issue by ekerrison** [astropy/astroquery#2014]
Hi, I've read the docs and astroquery.ned.get_table() can take one of the following as a table argument: ‘photometry’|’positions’|’diameters’|’redshifts’|’references’|’object_notes’.

Is it possible to query the other tables available in ned? in particular the 'Classifications' table?

**Comment by keflavich** [comment]
I don't think we have a resident NED expert, so I don't know the answer.  If anyone out there can help, please chime in!

**Comment by bsipocz** [comment]
No, at the moment only the following are supported:

```
        SEARCH_TYPE = dict(photometry='Photometry',
                           diameters='Diameters',
                           positions='Positions',
                           redshifts='Redshifts',
                           references='Reference',
                           object_notes='Notes')
```

**Comment by ConnorStoneAstro** [comment]
I am also interested in this feature especially 'Distances' and the 'Overview' table has a lot of good info (ie galaxy type). Also helpful would be 'Cross-IDs' and 'Galactic Extinctions'. I would be happy to help but I don't know how. I tried editing the "SEARCH_TYPE" dictionary manually, but that didn't work.

**Comment by kayhangultekin** [comment]
Today I needed this feature to get cross-ids for a list of sources. I have spent some time looking at the code so that I think I know how to do it, but I don't know when I'll have the time. The essential is that the NED query for a cross id of, e.g., NGC 4486 needs to look like

`http://ned.ipac.caltech.edu/cgi-bin/nph-objsearch?extend=no&of=xml_names&objname=NGC+4486`

One would need to update the `SEARCH_TYPE` dict to include 'Cross-IDs' (or whatever was needed), set the `of` field to `xml_names` and then (I think) everything will be handled properly as a VOTable.

**Comment by ConnorStoneAstro** [comment]
Today I learned that it is possible to construct tables with lots of useful properties using this url generator: https://ned.ipac.caltech.edu/forms/gmd.html

the resulting tables can be parsed with a bit of string manipulation and splitting on | characters. Perhaps with a bit of work this could be used to fill in some gaps. It has extinction, morphology, and distance information. Using python "requests" it's possible to pull the information. But I'm not sure how this integrates into VOTables or other astroquery stuff. (I'm a noob at scraping).
